src/contracts/SafeEntrypoint.sol

```diff
@@ -62,68 +52,49 @@ contract SafeEntrypoint is SafeManageable, ISafeEntrypoint {
     allowedActions[_actionContract] = false;
   }
 
+  /// @inheritdoc ISafeEntrypoint
+  function unqueueActionContract(address _actionContract) external isAuthorized {
+    queuedActions[_actionContract] = false;
+  }
```

Comment:
hmm, not, this should be `unqueueTransaction(bytes32 hash / uint nonce)`, and it should read from the ActionInfo struct the array of addresses that are being unqueued (data available now)

Reply:
Removed this method, action contracts are already unqueued in the unqueueAction method.